zinc/src/sbt-test/source-dependencies/scalac-options/incOptions.properties

```diff
@@ -0,0 +1 @@
+scalac.options = -Xfatal-warnings
```

Comment:
`+ignoredScalacOptions = -Xfatal-warnings` IMHO should be also added there
I would also add another options and change order - to have better coverage